### PR TITLE
[RU]MangaPoisk LICENSED notify deleted chapters

### PR DESCRIPTION
**Pull request by e-shl**
Checklist:

- [x] Updated `extVersionCode` value in `build.gradle` for individual extensions
- [x] Updated `overrideVersionCode` or `baseVersionCode` as needed for all multisrc extensions
- [x] Referenced all related issues in the PR body (e.g. "Closes #xyz")
- [x] Added the `isNsfw = true` flag in `build.gradle` when appropriate
- [x] Have not changed source names
- [x] Have explicitly kept the `id` if a source's name or language were changed
- [x] Have tested the modifications by compiling and running the extension through Android Studio

### CODE REVIEW

File: src/ru/mangapoisk/src/eu/kanade/tachiyomi/extension/ru/mangapoisk/MangaPoisk.kt [hunk 1]
```diff
@@ -150,6 +150,11 @@ class MangaPoisk : ParsedHttpSource() {
         else -> SManga.UNKNOWN
     }
     override fun fetchChapterList(manga: SManga): Observable<List<SChapter>> {
+        val document = client.newCall(GET("$baseUrl${manga.url}?tab=chapters", headers)).execute().asJsoup()
+        if (document.select(".text-md:contains(Главы удалены по требованию правообладателя)").isNotEmpty()) {
+            manga.apply { status = SManga.LICENSED }
```
Comment (alessandrojean) [inline src/ru/mangapoisk/src/eu/kanade/tachiyomi/extension/ru/mangapoisk/MangaPoisk.kt:155]: This method isn't supposed to change stuff inside the `SManga`.

**Review by alessandrojean — CHANGES_REQUESTED** [review]

File: src/ru/mangapoisk/src/eu/kanade/tachiyomi/extension/ru/mangapoisk/MangaPoisk.kt (hunk 1 above)
Comment (e-shl) [reply src/ru/mangapoisk/src/eu/kanade/tachiyomi/extension/ru/mangapoisk/MangaPoisk.kt:155]: > This method isn't supposed to change stuff inside the `SManga`.

Indeed, the basic tachi does not work. Remove the status assignment?

Comment (e-shl) [reply src/ru/mangapoisk/src/eu/kanade/tachiyomi/extension/ru/mangapoisk/MangaPoisk.kt:155]: it doesn't always work as i intended in j2k

Comment (alessandrojean) [reply src/ru/mangapoisk/src/eu/kanade/tachiyomi/extension/ru/mangapoisk/MangaPoisk.kt:155]: Please open an issue to report it in the J2K repo.

**Pull request merged**